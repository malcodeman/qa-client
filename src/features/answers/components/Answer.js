import React from "react";
import styled from "styled-components";

const Wrapper = styled.div`
  border-radius: 3px;
  border-bottom: 1px solid #e6e6e6;
  background-color: #fff;
  padding: 20px 0;
  display: flex;
  flex-direction: column;
`;

const Answer = props => {
  return (
    <Wrapper>
      <p>{props.body}</p>
    </Wrapper>
  );
};

export default Answer;
