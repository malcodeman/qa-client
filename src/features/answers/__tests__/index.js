import React from "react";
import renderer from "react-test-renderer";

import Answer from "../components/Answer";

test("Answer matches snapshot", () => {
  const component = renderer.create(
    <Answer
      body={"answer body"}
      createdAt={"2018-06-14T07:06:04.000Z"}
      author={"creator"}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
