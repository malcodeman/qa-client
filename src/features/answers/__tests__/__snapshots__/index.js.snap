// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answer matches snapshot 1`] = `
<div
  className="sc-bxivhb iBfkVx"
>
  <main
    className="sc-ifAKCX jPSUiy"
  >
    <div
      className="sc-bdVaJa bxcuYP"
    >
      <div
        className="sc-bwzfXH gfIHP"
        color="#858c93"
        direction="up"
        onClick={[Function]}
        size="14px"
      />
      <span
        className="sc-htpNat faymHB"
      />
      <div
        className="sc-bwzfXH gqmMyx"
        color="#858c93"
        direction="down"
        onClick={[Function]}
        size="14px"
      />
    </div>
    <p
      className="sc-EHOje kxbPyB"
    >
      answer body
    </p>
  </main>
  <footer
    className="sc-bZQynM eatfvr"
  >
    <div
      className="sc-gzVnrw caoKng"
    >
      <span>
        answered 
        17 minutes
         ago
      </span>
      <span>
        by 
        creator
      </span>
    </div>
  </footer>
</div>
`;
